GetDKAN/dkan
# Fix default content frequency values

The old key/value pairs for the frequency fields of the default content need to be updated.

## QA Steps

View the QA site and confirm that the key values for frequency on default datasets are ISO 8601 Repeating Duration values and a human readable term is displayed.

## Merge process

- [ ] Note if there is a special sequence for merging (such as PRs on other repos that need to be merged first) or other abnormalities to look out for.

## Reminders
